test/files/run/repl-trace-elided-more.scala

```diff
@@ -0,0 +1,5 @@
+import scala.tools.partest.{SessionTest, StackCleaner}
+
+// scala/bug#11945 print stack trace elided count correctly
```

Comment:
Of course, we'll never know, because the check file is redacted.

Reply:
LOL